### PR TITLE
pkp/pkp-lib#1231 Fix frontend display issues

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: classes/press/NewReleaseDAO.inc.php [hunk 1]
```diff
@@ -66,7 +66,10 @@ function getMonographsByAssoc($assocType, $assocId) {
 		$publishedMonographDao = DAORegistry::getDAO('PublishedMonographDAO');
 		while (!$result->EOF) {
 			list($monographId) = $result->fields;
-			$returner[] = $publishedMonographDao->getById($monographId);
```
Comment (asmecher) [inline classes/press/NewReleaseDAO.inc.php]: This change could be made more compactly and efficiently by moving it directly into the query above. I'll merge this (barring anything major) and make that change.

File: templates/frontend/objects/monograph_full.tpl [hunk 2]
```diff
@@ -167,7 +167,7 @@
 							{* Only one resource allowed per format, so mimic single-file-download *}
 							<div class="pub_format_{$publicationFormatId|escape} pub_format_remote">
 								<a href="{$publicationFormat->getRemoteURL()|escape}" target="_blank" class="remote_resource">
-									{translate key="payment.directSales.readRemotely" format=$publicationFormat->getLocalizedName()}
+									{$publicationFormat->getLocalizedName()}
```
Comment (asmecher) [inline templates/frontend/objects/monograph_full.tpl]: Needs `|escape` (I'll add this after merging)

**Comment by asmecher** [comment]
Replaced by https://github.com/pkp/omp/pull/264

File: classes/press/NewReleaseDAO.inc.php (hunk 1 above)
Comment (asmecher) [inline classes/press/NewReleaseDAO.inc.php]: (See: https://github.com/pkp/omp/commit/f3e31a4c6056de20e1653b830e8071efdd666e57#diff-922d67c7843051afca71401f17dfc4e2L54)

Comment (NateWr) [inline classes/press/NewReleaseDAO.inc.php]: :+1:

**Pull request closed without merging**